samples/nrf9160/nrf_cloud_mqtt_multi_service/src/main.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <logging/log.h>
+#include "application.h"
+#include "connection.h"
+#include "fota_support.h"
+
+LOG_MODULE_REGISTER(main, CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL);
+
+
+static void msg_thread_fn(void)
+{
+	/* Continuously consume (that is, send to NRF Cloud) queued-up outgoing messages */
+	while (true) {
+		(void)consume_device_message();
+	}
+}
+
+static void app_thread_fn(void)
+{
+	/* Wait for first connection before starting the application */
+	(void)await_connection(K_FOREVER);
+
+	/* Afterwards, allow application to run in this thread */
+	main_application();
+}
+
+static void conn_thread_fn(void)
+{
+	/* Continuously manage our connection */
+	manage_connection();
+}
+
+
+K_THREAD_DEFINE(app_thread, CONFIG_APPLICATION_THREAD_STACK_SIZE, app_thread_fn,
+		NULL, NULL, NULL, 0, 0, 0);
+
+K_THREAD_DEFINE(msg_thread, CONFIG_APPLICATION_THREAD_STACK_SIZE, msg_thread_fn,
```

Comment:
Your Kconfig defines a symbol for the message thread, but you are using the application thread size here.

Same for the connection thread.